tests/test_implementations/test_rustsim_implementation.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+
+from rustsim import get_intersection, jaccard_similarity, mrca_and_score
+
+from oaklib.datamodels.vocabulary import IS_A, PART_OF
+from oaklib.implementations.sqldb.sql_implementation import SqlImplementation
+from oaklib.resource import OntologyResource
+from oaklib.selector import get_adapter
+from tests import ENDOMEMBRANE_SYSTEM, INPUT_DIR, VACUOLE
+from tests.test_implementations import ComplianceTester
+
+DB = INPUT_DIR.joinpath("go-nucleus.db")
+
+
+class TestRustSimImplementation(unittest.TestCase):
+    """Implementation tests for Rust-based semantic similarity."""
+
+    def setUp(self) -> None:
+        """Set up"""
+        # Calling get_implementation_from_shorthand() alone fails on Windows
+        try:
+            oi = get_adapter(f"rustsim:sqlite:///{str(DB)}")
+        except FileNotFoundError:
+            oi = SqlImplementation(OntologyResource(slug=f"sqlite:///{str(DB)}"))
```

Comment:
I don't understand the reasoning here

This should test rustsim. It should fail fast if for some reason it can't connect